core/ir/aarch64/codec.txt

```diff
@@ -102,6 +103,7 @@
 -------------xxx------xxxxx-----  fpimm8     # floating-point immediate for vector fmov
 -------------xxx------xxxxx-----  imm8       # immediate from 16:18 and 5:10
 -------------xxxxxxxxxxxxxx-----  sysops     # immediate operands for SYS
+------------xxxx----------------  imm4_16    # Q register (0-15) if bit 30 is set, else D
```

Comment:
The comment looks like a copy-and-paste from `dq16_h_sz`. It's misleading in that bit 30 is not used. See comment below regarding `imm4_16` and `index_lhm`.

Reply:
done